packages/client/src/components/atoms/AddToCalendar/__tests__/AddToCalendar.test.tsx

```diff
@@ -0,0 +1,23 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import React from "react";
+import { screen } from "@testing-library/react";
+import * as reactRedux from "react-redux";
+
+import AddToCalendar from "../AddToCalendar";
+import { __addToCalendarButtonId__ } from "@/__testData__/testIds";
+import { renderWithRedux } from "@/__testUtils__/wrappers";
+
+const mockDispatch = jest.fn();
+jest.spyOn(reactRedux, "useDispatch").mockImplementation(() => mockDispatch);
+
+describe("Add To Calendar", () => {
+  describe("Smoke Test", () => {
+    test("should render component", () => {
+      renderWithRedux(<AddToCalendar />);
+      screen.getByTestId(__addToCalendarButtonId__);
+    });
+  });
+});
```

Comment:
This file is still unnecessary:
- as I understand the complete functionality is tested with cypress
- as much as I believe in unit tests (for TDD), even though the feature is later tested in e2e, here you're just testing that something exists in screen, and it's not even something visible, but rather a test id created and assigned only to test it being assigned

Reply:
In your previous comment you mentioned to leave it be if I was planning to add more tests, which I am. In my next PR `feature/updatable_calendar_events` I write another test.
I could delete the file from this PR if that makes more sense.